src/rules/no-unused-modules.js

```diff
@@ -201,6 +203,58 @@ const newNamespaceImportExists = specifiers =>
 const newDefaultImportExists = specifiers =>
   specifiers.some(({ type }) => type === IMPORT_DEFAULT_SPECIFIER)
 
+const fileIsInPkg = file => {
+  const { path, pkg } = readPkgUp.sync({cwd: file, normalize: false})
+  const basePath = dirname(path)
+
+  const checkPkgFieldString = pkgField => {      
+    if (join(basePath, pkgField) === file) {
+        return true
+      }
+  }
+
+  const checkPkgFieldObject = pkgField => {    
+      const pkgFieldFiles = Object.values(pkgField).map(value => join(basePath, value))
```

Comment:
`Object.values` is not available on all supported nodes; please use https://npmjs.com/object.values instead

Reply:
I would rather use `_.values(object)`, if it is okay for you. Lodash already is a project dependency, with object.values we would add a new dependency.

Reply:
I'd prefer to use the standard method; adding a dependency isn't a problem.